tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -383,6 +394,27 @@ describe(__filename, () => {
       );
     });
 
+    it('fetches with null scores when empty', () => {
+      const addon = { ...fakeAddon };
+      loadAddon(addon);
+      const dispatch = sinon.stub(store, 'dispatch');
+
+      const root = render({
+        // Set up a location where ?score= is not present in the URL.
+        location: createFakeLocation({ query: {} }),
+        params: { addonSlug: addon.slug },
+      });
+
+      sinon.assert.calledWith(
+        dispatch,
+        _fetchReviews({
+          addonSlug: addon.slug,
+          errorHandlerId: root.instance().props.errorHandler.id,
```

Comment:
nit: I've seen this both ways (for example in above test) but what about setting this above `const errorHandler = createStubErrorHandler()` and then using `errorHandler.id` here?

Reply:
I like this way better because it's less code and thus more readable. You don't have to set up a variable and you don't have to pass that variable to `render()`.